web/satellite/src/api/usage.ts

```diff
@@ -3,43 +3,39 @@
 
 import apollo from '@/utils/apollo';
 import gql from 'graphql-tag';
-import { RequestResponse } from '@/types/response';
+import { ProjectUsage, UsageApi } from '@/types/usage';
```

Comment:
imports sort order

Reply:
it is job for tslint 
https://palantir.github.io/tslint/rules/ordered-imports/